Jenkinsfile

```diff
@@ -243,6 +243,32 @@ pipeline {
     }
 
     stages {
+        stage('check_for_changed_files') {
+          steps {
+            script {
+	      def all_files = new ArrayList()
+
+              def changeLogSets = currentBuild.changeSets
+              for (int i = 0; i < changeLogSets.size(); i++) {
+                def entries = changeLogSets[i].items
+                for (int j = 0; j < entries.length; j++) {
+                  def entry = entries[j]
+                  def files = new ArrayList(entry.affectedFiles)
+                  for (int k = 0; k < files.size(); k++) {
+                    def file = files[k]
+                    println file.path
+                    all_files << file.path
+                  }
+                }
+              }
+		  
+	      if (all_files.length == 1 && all_files[0] == 'CHANGELOG.md') {
+	        error "Only CHANGELOG.md was modified"
```

Comment:
We _probably_ don't need a rebuild for `README.md` (and other markdown files), either?  Up to you; maybe there are situations where we do…